<launch>
    <node pkg="slam" type="grid_position_transform.py" name="grid_position_transform" output="screen"/>
    <node pkg="slam" type="explore_frontiers.py" name="explore_frontiers" output="screen"/>
    <node pkg="slam" type="motion_control.py" name="motion_control" output="screen"/>
    
    
    <!-- Launch the spwan_robot.launch file from the spot_config package -->
    <include file="$(find slam)/launch/mapping.launch" /> 

    <!-- <include file="$(find champ_teleop)/launch/teleop.launch" launch-prefix="xterm -e" /> -->
</launch>
